packages/types/src/wsTypes.ts

```diff
@@ -4,45 +4,43 @@ export interface Room {
   members: Member[];
 }
 
-export enum WebSocketRequestType {
-  createGroupSession,
-  joinGroupSession,
-  leaveGroupSession,
-  sendData,
-}
+export type WebSocketRequestType =
+  | 'create-group-session'
+  | 'join-group-session'
+  | 'leave-group-session'
+  | 'send-data';
 
-export enum WebSocketResponseType {
-  createdGroupSession,
-  failedToCreateGroupSession,
-  joinedGroupSession,
-  failedToJoinGroupSession,
-  memberJoinedGroupSession,
-  memberLeftGroupSession,
-  dataReceived,
-}
+export type WebSocketResponseType =
+  | 'created-group-session'
+  | 'failed-to-create-group-session'
+  | 'joined-group-session'
+  | 'failed-to-join-group-session'
+  | 'member-joined-group-session'
+  | 'member-left-group-session'
+  | 'data-received';
```

Comment:
Replaced the enums with types early in my please-make-this-compile journey – TypeScript enums are not known for being the go-to, so I think it's nice to leave it as types, even though it might be a minor downgrade on the code-exploration front.

Reply:
Might be possible to do this in combination of constants?
I can probably try in another PR. Not super important